modules/dr-restoring-cluster-state.adoc

```diff
@@ -73,7 +73,7 @@ The output of this command should be empty. If it is not empty, wait a few minut
 +
 [source,terminal]
 ----
-[core@ip-10-0-154-194 ~]$ sudo crictl ps | grep kube-apiserver
+[core@ip-10-0-154-194 ~]$ sudo crictl ps | grep kube-apiserver | grep -v operator
```

Comment:
optional nit we can use explicit labels with crictl vs nested grep.

```
crictl ps --label io.kubernetes.container.name=kube-apiserver
crictl ps --label io.kubernetes.container.name=etcd
```

Reply:
or --name with a simple regex pattern

```
crictl ps --name ^etcd$
```

Reply:
@hexfusion Well, we are not just looking for `kube-apiserver` to stop, but all other containers that are part of the the kube-apiserver pod.

Reply:
I'm going to merge this since we know that it works and is QE approved, but let me know if we want to update these commands and I can do that in a separate PR.